config/config.go

```diff
@@ -68,6 +68,41 @@ func New(uri string, requests, concurrency int, requestTimeout, globalTimeout ti
 	}
 }
 
+// Validate returns the config and a not nil ErrInvalid if any of the fields provided by the user is not valid
+func (cfg Config) Validate() error {
+	inputErrors := []error{}
+
+	_, err := url.ParseRequestURI(cfg.Request.URL.String())
+	if err != nil {
+		inputErrors = append(inputErrors, fmt.Errorf("-url: %s is not a valid url", cfg.Request.URL.String()))
+	}
+
+	if cfg.RunnerOptions.Requests < 0 {
+		if cfg.RunnerOptions.Requests != -1 {
```

Comment:
We can avoid that extra nesting using `&&`, also I think `0` should be an invalid values for `requests`:
```suggestion
	if cfg.RunnerOptions.Requests < 1 && cfg.RunnerOptions.Requests != -1 {
```

Reply:
I first used `&&`, but `gocognit` was complaining that it was too complex :(
It is right though, 0 should also be invalid.

Reply:
Well if `gocognit` considers a single `&&` more complex than a nested `if` then it doesn't do its job correctly 🤷‍♂️ 
I suggest we shut it down and remove it from `.golangci.yml`.

Reply:
I have used `//nolint `on the method for the moment, if we run into this problem again, we can remove `gocognit` completely...